third_party/nvidia/backend/compiler.py

```diff
@@ -242,6 +260,10 @@ def make_llir(src, metadata, options, capability):
             paths = [path for (name, path) in options.extern_libs]
             llvm.link_extern_libs(llvm_mod, paths)
 
+        proc = 'sm_90a' if capability == 90 else f'sm_{capability}'
```

Comment:
if it makes sense, I'd do this right after `llvm.to_module` at L250 given it's typically better to have the data layout before any analysis transformation on the llvm module. (This place is not far from that but just to minimize the chance we add something in the middle..)